tests/integration/steps_submit_equivocation_proposal.go

```diff
@@ -21,7 +61,7 @@ func stepsSubmitEquivocationProposal(consumerName string, propNumber uint) []Ste
 					ValPowers: &map[validatorID]uint{
 						validatorID("alice"): 509,
 						validatorID("bob"):   500,
-						validatorID("carol"): 495,
+						validatorID("carol"): 0,
```

Comment:
why is this change introduced?

Reply:
Because double signing was added to happy path, since this is in happy path it needs to reflect the double sign.